mlflow/server/js/src/experiment-tracking/components/experiment-page/components/traces-v3/TracesV3Toolbar.tsx

```diff
@@ -40,6 +50,29 @@ export const TracesV3Toolbar = ({ viewState, sessionId }: { viewState: string; s
           <Typography.Title level={3} withoutMargins>
             {sessionId}
           </Typography.Title>
+          <Tooltip
+            componentId="mlflow.chat-sessions.copy-session-id"
+            content={
+              showCopyTooltip ? (
+                <FormattedMessage defaultMessage="Copied!" description="Tooltip after copying session ID" />
+              ) : (
+                <FormattedMessage defaultMessage="Copy session ID" description="Tooltip for copy session ID button" />
+              )
+            }
+            open={showCopyTooltip ? true : undefined}
+          >
+            <CopyIcon
+              onClick={handleCopySessionId}
+              css={{
+                cursor: 'pointer',
+                color: theme.colors.textSecondary,
+                fontSize: 16,
+                '&:hover': {
+                  color: theme.colors.textPrimary,
+                },
+              }}
+            />
+          </Tooltip>
```

Comment:
While we're here, add a copy button for session ID to the chat sessions UI:

<img width="774" height="267" alt="Screenshot 2025-11-23 at 9 51 21 PM" src="https://github.com/user-attachments/assets/c6afb413-82df-4dda-9c92-944f9f9612bb" />